src/ui/units/workbooks/components/EntryActions/EntryActions.tsx

```diff
@@ -34,16 +35,20 @@ export const EntryActions = ({
 }: EntryActionsProps) => {
     const {useAdditionalWorkbookEntryActions} = registry.workbooks.functions.getAll();
 
+    const isConnection = entry.scope === EntryScope.Connection;
+    const isNotFileConnection =
```

Comment:
The same issue here

Reply:
ok, added // eslint-disable-next-line no-negated-condition